### PR TITLE
UIImageButton & UI changes

### PR DESCRIPTION
**Pull request by MikhailMCraft**
Changed UI rectangle drawing and added a `InteractionBoxRelative` parameter that adds to `InteractionBox` accordingly.

### CODE REVIEW

File: Content/Content.mgcb [hunk 1]
```diff
@@ -38,6 +38,18 @@
...
+/importer:TextureImporter
+/processor:TextureProcessor
+/processorParam:ColorKeyColor=255,0,255,255
+/processorParam:ColorKeyEnabled=True
+/processorParam:GenerateMipmaps=False
+/processorParam:PremultiplyAlpha=True
+/processorParam:ResizeToPowerOfTwo=False
+/processorParam:MakeSquare=False
+/processorParam:TextureFormat=Color
+/build:Assets/thing.png
```
Comment (RighteousRyan1) [inline Content/Content.mgcb:51]: Why is this still here...

File: GameContent/WPTR.cs [hunk 2]
```diff
@@ -18,12 +18,22 @@ public class WPTR
 
         private static UIElement lastElementClicked;
 
-        private static UIParent parent;
+        private struct UIParents
```
Comment (RighteousRyan1) [inline GameContent/WPTR.cs:21]: Can you put all the stuff related to the pause menu into it's own separate class?

File: Internals/UI/OuRectangle.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.Xna.Framework;
+
+namespace WiiPlayTanksRemake.Internals.UI
+{
+    public struct OuRectangle
```
Comment (RighteousRyan1) [inline Internals/UI/OuRectangle.cs:5]: Rename this to something like RectangleF. OuRectangle is too.... meme-y?

File: TankGame.cs [hunk 4]
```diff
@@ -70,9 +72,6 @@ protected override void Update(GameTime gameTime)
         {
             Input.HandleInput();
 
-            foreach (var bind in Keybind.AllKeybinds)
```
Comment (RighteousRyan1) [inline TankGame.cs:73]: why was this removed?

Comment (MikhailMCraft) [reply TankGame.cs:73]: Moved it to `WPTR.Update()`

File: Content/Content.mgcb (hunk 1 above)
Comment (MikhailMCraft) [reply Content/Content.mgcb:51]: Removed

File: GameContent/WPTR.cs [hunk 5]
```diff
@@ -93,10 +82,12 @@ internal static void Draw()
...
-            UIParents.PauseMenuParent = new();
-            UIElements.PauseReturnButton = new("Return", TankGame.Fonts.Default, Color.Gray, Color.White, 1.5f);
-            UIElements.PauseReturnButton.InteractionBoxRelative = new OuRectangle(0.35f, 0.25f, 0.3f, 0.1f);
-            UIParents.PauseMenuParent.AppendElement(UIElements.PauseReturnButton);
+            UI.PauseMenu.MenuParent = new();
+            UI.PauseMenu.UIElements.PauseButtonReturn = new("Return", TankGame.Fonts.Default, Color.Gray, Color.White, 1.5f)
+            {
+                InteractionBoxRelative = new OuRectangle(0.35f, 0.25f, 0.3f, 0.1f)
+            };
+            UI.PauseMenu.MenuParent.AppendElement(UI.PauseMenu.UIElements.PauseButtonReturn);
```
Comment (RighteousRyan1) [inline GameContent/WPTR.cs:90]: All of this can be moved to `PauseMenu.cs`.
